README.md

```diff
@@ -25,9 +25,23 @@ Data about food is of public interest and has to be open. The complete database
 
 ## Contributing
 
-The project is initially started by [](https://github.com/), other contributors include:
+Any help is welcome as long as you don't break the continuous integration. Fork
+the repository and push directly to master.
+
+Maintainers:
+
+- [Frank Rousseau](https://github.com/frankrousseau)
 - [Pierre Slamich](https://github.com/teolemon)
 
+Contributors:
+
+- [Abunav Bhargavah](https://github.com/Anubhav-Bhargavah)
```

Comment:
[Anubhav Bhargava]#(https://github.com/Anubhav-Bhargava)

Reply:
Sorry for the mistake